docs/rules/no-import-test-files.md

```diff
@@ -0,0 +1,60 @@
+# Ensure no tests are imported anywhere
+
+This rule will verify that you don't import any test files. It will consider the root of the project to be the closest folder containing a `package.json` file, and will not do anything if it can't find one. Test files in `node_modules` will not be linted as they are ignored by ESLint.
+
+Note that this rule will not be able to warn correctly if you use AVA by specifying the files in the command line ( `ava "lib/**/*.test.js"` ). Prefer configuring AVA as described in the link above.
+
+## Fail
+
+```js
+// File: src/index.js
+// Invalid because *.test.js is considered as a test file.
+import tests from './index.test.js';
+
+// File: src/index.js
```

Comment:
It's confusing that you have multiple file examples in the same code block. They should be separate code blocks.